shim/src/realmFacade.js

```diff
@@ -99,13 +96,20 @@ export function buildChildRealm(BaseRealm) {
     }
   }
 
-  defineProperty(Realm.prototype, 'toString', {
+  defineProperty(Realm, 'toString', {
```

Comment:
do you control the Function toString value such that `Function.toString.call` will respect this property? What happens if i `delete Realm.toString; String(Realm)`?

Reply:
It will print the source code from the shim, which thereby fails to perfectly emulate the specified behavior. For now, I don't think we should fix this, but rather document it as a low priority open issue.

This all gets more interesting with the functions we write for purposes of stringifying and then evaluating in another realm. Separately from this PR, we should examine attacks by forced divergences among ways to stringify those functions.

Reply:
Correct, this is not done for security but strictly for ergonomics, to help with debugging.

Reply:
Overriding Function.prototype.toString would provide the security though, no?